manim/mobject/text/code_mobject.py

```diff
@@ -156,7 +156,8 @@ def construct(self):
     # For more information about pygments.lexers visit https://pygments.org/docs/lexers/
     # from pygments.lexers import get_all_lexers
     # all_lexers = get_all_lexers()
-    styles_list = list(get_all_styles())
+    # styles_list = list(get_all_styles())
```

Comment:
```suggestion
```
I don't think there's a need for this comment, as it doesn't elaborate much?

Reply:
Fine by me. I was just repeating what I saw with the other commented lines before it.